.vib/moodle/vib-publish.json

```diff
@@ -22,7 +22,7 @@
           "url": "{SHA_ARCHIVE}",
           "path": "/bitnami/moodle"
         },
-        "runtime_parameters": "Im1vb2RsZVBhc3N3b3JkIjogImJpdG5hbWkxIgoibW9vZGxlVXNlcm5hbWUiOiAidXNlciIKInNlcnZpY2UiOgogICJwb3J0IjogODAKICAidHlwZSI6ICJMb2FkQmFsYW5jZXIi",
+        "runtime_parameters": "bW9vZGxlVXNlck5hbWU6IHVzZXIKbW9vZGxlUGFzc3dvcmQ6IENvbXBsaWNhdGVkUGFzc3dvcmQxMjMhNApzZXJ2aWNlOgogIHR5cGU6IExvYWRCYWxhbmNlcgogIHBvcnRzOgogICAgaHR0cDogODAKbWFyaWFkYjoKICBhdXRoOgogICAgZGF0YWJhc2U6IGJpdG5hbWlfbW9vZGxlCiAgICB1c2VybmFtZTogYm5fbW9vZGxlCg==",
```

Comment:
As discussed yesterday, we are now modifying the default values of the parameters used by default so we can adequately test the chart integration. You will need to at least change the following parameters:
 
- asset and database credentials
- podSecurityContext.xxx/containerSecurityContext.xxxx
- containerPorts.xxx and service.ports.https

Except for a test for https as you did for apache, the other changes won't require an explicit test. This is because having the app deploy correctly is tests enough to verify these non-default values work properly.